pkg/util/machines/machines.go

```diff
@@ -0,0 +1,77 @@
+package machines
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	mapiv1 "github.com/openshift/cluster-api/pkg/apis/machine/v1beta1"
+	healthcheckingv1alpha1 "github.com/openshift/machine-api-operator/pkg/apis/healthchecking/v1alpha1"
+	"github.com/openshift/machine-api-operator/pkg/util/conditions"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// IsMachineHealthy returns true if the the machine is running and machine node is healthy
+func IsMachineHealthy(c client.Client, machine *mapiv1.Machine) bool {
```

Comment:
this might/might not disagree with the MHC configmap

Reply:
yep I wanted to adapt it in the future PR